src/service/recording_service.py

```
@@ -0,0 +1,29 @@
+from src.ai.speech2text import SpeechToText
+from fastapi import HTTPException
+import threading
```

Ruff (0.8.2)

`threading` imported but unused

Remove unused import: `threading`

(F401)